tests/integ/scala/test_dataframe_reader_suite.py

```diff
@@ -332,6 +332,51 @@ def test_to_read_files_from_stage(session, resources_path, mode):
         session.sql(f"DROP STAGE IF EXISTS {data_files_stage}")
 
 
+@pytest.mark.parametrize("mode", ["select", "copy"])
+def test_to_read_csv_from_local(session, resources_path, mode):
+
+    test_files = TestFiles(resources_path)
+
+    reader = get_reader(session, mode)
+
+    try:
+        df = (
+            reader.schema(user_schema)
+            .option("compression", "auto")
+            .csv(test_files.test_file_csv)
+        )
+        res = df.collect()
+        res.sort(key=lambda x: x[0])
+        assert res == [
+            Row(1, "one", 1.2),
+            Row(2, "two", 2.2),
+        ]
+    finally:
+        pass
+
+
+# @pytest.mark.parametrize("mode", ["select", "copy"])
+# def test_to_read_csv_from_local_directory(session, resources_path, mode):
+#     test_files = TestFiles(resources_path)
+#
+#     reader = get_reader(session, mode)
+#
+#     try:
+#         df = (
+#             reader.schema(user_schema)
+#                 .option("compression", "auto")
+#                 .csv(test_files)
+#         )
+#         res = df.collect()
+#         res.sort(key=lambda x: x[0])
```

Comment:
You don't need sorting here, right? If you need, `Utils.check_answer` provides such an argument.